Games/types/Mafia/roles/Village/President.js

```diff
@@ -4,6 +4,6 @@ module.exports = class President extends Role {
   constructor(player, data) {
     super("President", player, data);
     this.alignment = "Village";
-    this.cards = ["VillageCore", "WinWithVillage", "KillAlignedOnDeath"];
+    this.cards = ["VillageCore", "WinWithVillage", "KillVillageOnDeath"];
```

Comment:
our version of pres has been reworked to "linchpin"

can kill mafia on death, kill cult on death depending on alignment